components/UserButton.vue

```diff
@@ -1,23 +1,26 @@
 <template>
   <v-menu offset-y>
     <template #activator="{ on, attrs }">
-      <v-btn icon v-bind="attrs" v-on="on">
-        <v-avatar>
-          <v-img
-            v-if="userImage"
-            :src="userImage"
-            :alt="$auth.user.Name"
-          ></v-img>
-          <v-icon v-else dark>mdi-account</v-icon>
-        </v-avatar>
-      </v-btn>
+      <div class="d-flex" v-bind="attrs" v-on="on">
+        <div class="mr-4">
+          <v-btn icon>
+            <v-avatar>
```

Comment:
Not essential, but I think it looks slightly better with a background color on the avatar icon

Reply:
Indeed, I think it was removed when moving stuff around between the two user picture components to make a single one. I didn't notice it due to having an avatar set at the time.

It's fixed in 53015bd :sweat_smile: